scripts/test.ps1

```diff
@@ -22,7 +22,7 @@ $vstest = "& '$vstest'"
 
 $AdapterPath = Join-Path $ProjectDir "artifacts\$configuration\net451"
 
-$command = "$vstest --TestAdapterPath:$AdapterPath --inisolation"
+$command = "$vstest --TestAdapterPath:$AdapterPath"

```

Comment:
>$AdapterPath [](start = 38, length = 12)

any reaon to remove the isolation flag ?

Reply:
The console logs don't show up in the console if the tests are running in isolation